define_burnup/info.yml

```diff
@@ -0,0 +1,13 @@
+---
+uid: define_burnup
+title: Define nuclear fuel burnup
+prerequisites:
+  - specific_thermal_power
+learning_objectives:
+  - define the concept of nuclear fuel burnup
+references:
+  - None
+assessments: 
+  - burnup_comparison
+...
```

Comment:
I'm not sure these dots are necessary. That said, I'm not sure they're not necessary.